examples_tests/09.ColorSpaceTest/main.cpp

```diff
@@ -96,24 +96,32 @@ bool ReadLine(io::IReadFile* file, std::string &line)
 void testImage(const std::string& path, IrrlichtDevice* device, video::IGPUMeshBuffer* fullScreenTriangle)
 {
 	os::Printer::log("Reading", path);
+	
 	auto& assetMgr = device->getAssetManager();
-	video::IVideoDriver* driver = device->getVideoDriver();
+	auto* driver = device->getVideoDriver();
 
 	asset::ICPUTexture* cputex[1] = { static_cast<asset::ICPUTexture*>(assetMgr.getAsset(path, {})) };
 	
 	if (cputex[0])
 	{
-		io::path filename,extension ;
+		io::path filename, extension;
 		core::splitFilename(path.c_str(), nullptr, &filename, &extension);
 		filename += "."; filename += extension;
-
+		
+		bool writeable = (extension != "dds") && (extension != "bmp");
```

Comment:
so you're nuking the BMP writer, correct?

Reply:
Yes, I don't really see a reason why one would have a reason to write to BMP. Give me the final signal and I'll wipe the writer from the codebase.